inventory/def.cf

```diff
@@ -0,0 +1,26 @@
+bundle agent inventory_def_file_control
+{
+  vars:
+  # Each input should be guarded with both an enable and !disable class so that
```

Comment:
Rather that classes as we did in 3.6, maybe the inventory control should be through a central slist.  Then it's easy to control its contents from a central UI in MP, and there's no need for enable/disable classes.